### PR TITLE
Video settings

### PR DESCRIPTION
**Pull request by nik0**
Video settings including video format selection thanks to @ixisandyr

### CODE REVIEW

**Comment by saipsa** [comment]
Hey @nik0 I cannot find video format selection options in the web interface. Is it only available via command line?

File: firmware_mod/www/cgi-bin/status.cgi [hunk 1]
```diff
@@ -543,6 +578,81 @@ cat << EOF
     </div>
 </div>
 
+<!-- Audio In (Mic) -->
```
Comment (jmtatsch) [inline firmware_mod/www/cgi-bin/status.cgi]: Looks like a duplicate of line 688. Which one is better?

File: firmware_mod/www/cgi-bin/status.cgi [hunk 2]
```diff
@@ -543,6 +578,81 @@ cat << EOF
...
+            <p class="control">
+                <input id="audioinSubmit" class="button is-primary" type="submit" value="Set" />
+            </p>
+        </form>
+    </div>
+</div>
+
 <!-- Bitrate -->
 <div class='card status_card'>
     <header class='card-header'><p class='card-header-title'>Video quality</p></header>
```
Comment (jmtatsch) [inline firmware_mod/www/cgi-bin/status.cgi]: This can be removed in favour of the new 'Video Settings' tab?

**Comment by nik0** [comment]
@saipsa,
No there is a menu in the settings (I re-use the tab with the video size)

**Pull request merged**